sunkit_pyvista/plotter.py

```diff
@@ -59,6 +61,31 @@ def _coords_to_xyz(self, coords):
                                 coords.y.to_value(R_sun),
                                 coords.z.to_value(R_sun)))
 
+    def set_camera_coordinates(self, coord):
+        """
+        Sets the inital camera position of the rendered plot.
+
+        Parameters
+        ----------
+        coords : `astropy.coordinates.SkyCoord`
+            Coordinates of the camera.
+        """
+        camera_position = self._coords_to_xyz(coord)
+        pos = tuple(camera_position[0])
+        self.plotter.camera.position = pos
+
+    def rotate_camera(self, angle: u.deg = None):
```

Comment:
I think it would be better to make this a method that sets the camera rotation, instead of adding (or subtracting) a given amount of rotation. This way the rotation is always relative to the solar north pole when specified. Thoughts?

Reply:
Yes, that was what I initially thought of, but looking the comment I made above, the default map produces a plot like :
![image](https://user-images.githubusercontent.com/50923538/120924950-c885c900-c6c5-11eb-8588-621617f3c0d5.png)

Reply:
I think what we should do here is
- Specify that the input to our `rotate_camera` method is relative to the solar north pole.
- Internally subtract 90 degrees from the value given to set the `plotter.camera.roll` property correctly
- Call this method as `self.rotate_camer(0*u.deg)` in the plotter `__init__` to set the default starting camera position to solar north up

I think we should also rename `rotate_camera` to `set_camera_rotation`.

Reply:
Sure, that's what I assumed we'd do but the issue with that is the fact that different coordinate frames have different initial value set right? So, if we're rotating relative to the solar north pole, we'd just have to add to the current rotation because it's always pointing up anyway, thoughts?

Reply:
Ah, I forgot that the plotter could have different coordinate systems set. In that case the rotation should be relative to the z-axis of the coordinate system, which for the default coordinate frame is the solar north pole. I don't think this means that we want to be adding to the rotation each time though.

Reply:
I'll play around with this a bit tonight and see what the results are. I didn't find rotation wrt a single axes in the docs. I'll update this PR in a bit, thanks for the help :)